tests/integration/test_charm.py

```diff
@@ -318,40 +241,13 @@ async def test_anonymize_user(
     )
     await action_register_user.wait()
     assert action_register_user.status == "completed"
```

Comment:
Once MAS is set, is there another way of checking/calling the Synapse API? This would allow us to validate whether the action succeeded.

Reply:
We have the dedicated MAS admin api for that. But I don't think we can reuse the synapse API client as this API uses OIDC spec for authentication / authorization. I think we can leave it like this for now since doing that would be quite an involved task.